packages/manager/modules/overthebox/src/index.js

```diff
@@ -1,26 +1,44 @@
 import angular from 'angular';
-
-import '@ovh-ux/manager-core';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
-const moduleName = 'ovhManagerOtb';
-
-angular.module(moduleName, [
-  'ui.router',
-  'oc.lazyLoad',
-  'ovhManagerCore',
-])
-  .config(/* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('overTheBox.**', {
-      url: '/overTheBox/:serviceName',
-      lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
-
-        return import('./overTheBox.component')
-          .then(mod => $ocLazyLoad.inject(mod.default || mod));
+import '@ovh-ux/manager-core';
+
+import { ListLayoutHelper } from '@ovh-ux/ng-ovh-telecom-universe-components';
+
+import overTheBox from './overTheBox';
+
+import component from './overtheboxes.component';
+
+const moduleName = 'ovhManagerOverTheBoxes';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'ovhManagerCore',
+    'oc.lazyLoad',
+    overTheBox,
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('overTheBoxes', {
+      url: '/overTheBox',
+      abstract: true,
+    });
+
+    $stateProvider.state('overTheBoxes.index', {
+      url: `?${ListLayoutHelper.urlQueryParams}`,
+      component: 'ovhManagerOverTheBoxes',
+      params: ListLayoutHelper.stateParams,
+      resolve: {
+        apiPath: () => '/overTheBox',
+        ...ListLayoutHelper.stateResolves,
+        getOvertheboxLink: /* @ngInject */ $state => otb => $state.href('overTheBoxes.overTheBox.details', {
```

Comment:
Could you please prevent using mangled variable name?

```suggestion
        getOvertheboxLink: /* @ngInject */ $state => overTheBox => $state.href('overTheBoxes.overTheBox.details', {
```

Reply:
Eslint grumping because something is already named that way in the upper scope :) (we'll check if it makes sense to change the name